jfrog-cli/docs/common/help.go

```diff
@@ -16,4 +16,8 @@ const GlobalEnvVars string = `	JFROG_CLI_LOG_LEVEL
 	JFROG_CLI_HOME_DIR
 		[Default: ~/.jfrog]
 		Defines the JFrog CLI home directory path.
+	
+	JFROG_CLI_SHOW_UPDATE
```

Comment:
I suggest renaming it to JFROG_CLI_SHOW_VERSION_UPDATE.

Reply:
Thanks for the feedback, totally agree with changing the name.